src/passport-saml/saml.ts

```diff
@@ -147,6 +147,7 @@ class SAML {
         ctorOptions.identifierFormat === undefined
           ? "urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress"
           : ctorOptions.identifierFormat,
+      wantAssertionsSigned: ctorOptions.wantAssertionsSigned ?? false,
```

Comment:
If this is supposed to be boolean, then we should probably force a coercion here with `!!`.

Reply:
I think the changes should be done in `README.md` in another PR.